src/sst/core/testingframework/sst_unittest.py

```diff
@@ -48,6 +48,15 @@ class SSTTestCase(unittest.TestCase):
         This class is derived from Python's unittest.TestCase which provides an
         basic resource for how to develop tests for this frameworks.
     """
+    module_init = 0
+    module_sema = threading.Semaphore()
+
+    @classmethod
```

Comment:
@gvoskuilen  and I discussed this and we think that since this function doesn't do anything useful we should either remove it or have it be empty and have documentation on how to properly do the semaphore if it's needed.  I would only leave the empty function if it's required to avoid errors.